pkg/cloudprovider/aws/instancetypes.go

```diff
@@ -76,7 +78,13 @@ func (p *InstanceTypeProvider) Get(ctx context.Context, provider *v1alpha1.AWS)
 	}
 	var result []cloudprovider.InstanceType
 	for _, i := range instanceTypes {
-		result = append(result, NewInstanceType(ctx, i, provider, p.createOfferings(i, instanceTypeZones[aws.StringValue(i.InstanceType)])))
+		instanceType, err := NewInstanceType(ctx, i, p.pricingProvider, provider, p.createOfferings(i, instanceTypeZones[aws.StringValue(i.InstanceType)]))
+		if err != nil {
+			// this can occur often in an isolated VPC with a stale price list, so we only report it at the debug level
+			logging.FromContext(ctx).Debugf("skipping instance type %s due to %s", aws.StringValue(i.InstanceType), err)
```

Comment:
IIUC, printing the string value of an error results in a pretty human unfriendly string. Here's the pattern we use broadly in the code base. 
```suggestion
			logging.FromContext(ctx).Debugf("skipping instance type %s, %s", aws.StringValue(i.InstanceType), err.Error())
```

Reply:
printf calls err.Error internally automatically for  %v, %s, %x, %X and %q see https://github.com/golang/go/blob/668041ef66ddafffccf1863e6180b83ea1ad30c9/src/fmt/print.go#L651